apps/command-palette-wp-admin/src/use-commands.ts

```diff
@@ -14,10 +14,11 @@ const waitForElementAndClick = ( selector: string, attempt = 1 ) => {
 export const useCommandsWpAdmin = (): Command[] => {
 	// Only override commands that need a specific behavior for WP Admin.
 	// Commands need to be defined in `packages/command-palette/src/commands.tsx`.
-	COMMANDS.getHelp.callback = ( { close }: CommandCallBackParams ) => {
```

Comment:
@mmtr, I noticed this side effect, but couldn't find any signs of being intentional or any relevant to the current behavior. However, I just wanted to check with you to confirm as it will be gone after switching to the `useCommands()` hook.

Reply:
It seems to me that things will continue to work as they were before because we're still overriding the callback of the "Get help" command for WP Admin (although I couldn't test it because of error I mentioned earlier).

Reply:
Confirmed it still works as expected.